core/src/components/picker/picker.ios.scss

```diff
@@ -18,5 +18,5 @@
 }
 
 :host .picker-highlight {
-  background: var(--highlight-background, var(--ion-color-step-150, var(--ion-background-color-step-150, #eeeeef)));
+  background: var(--highlight-background, var(--ion-components-ion-picker-highlight-bg));
```

Comment:
This feels a bit redundant. Should we get rid of `--highlight-background` or leave it in?